wallet/purescript/Main.hs

```diff
@@ -1,109 +0,0 @@
-module Main
```

Comment:
Spurious bin file above ^^

Reply:
That's why it has been removed by this PR ;)